pilot/lib/pilot/build_manager.rb

```diff
@@ -410,7 +410,7 @@ def distribute_build(uploaded_build, options)
       # This is where we could add a check to see if encryption is required and has been updated
       uploaded_build = set_export_compliance_if_needed(uploaded_build, options)
 
-      if options[:groups] || options[:distribute_external]
+      if options[:distribute_external]
```

Comment:
This _totally_ makes sense but this would break existing users 😬 `: distribute_external` defaults to `false` which means if any users only set `:groups` then their existing behavior would break 😔

 I'm not a huge fan of this existing behavior but I don't think we can change that...

I wonder if we need to look at the groups that were specified and **not submit** for review if any of them are internal 🤔 Thoughts?

Reply:
I missed this point, you are totally right 🙌
However, not submit for review if any of them are internal might not be the best solution. Users can still provide for example two internal groups and one external, and in that case they would want to send the app for a review. I see two possibilities here:

 1. We can check the groups and only **not submit** the app for review if every group they provided is internal
 2. We add a new parameter to pilot, something like `submit_beta_review` which defaults to `true`, so we keep the existing behaviour. In case the user sets it explicitly to false, the app would not be sent to reiview.

Personally I prefer the second solution, as it gives more control to the users. There might be use cases when they still want to send the app for a review even though all the groups they provided are internal ones. WDYT?

Reply:
@horvathtamasattila I like the `submit_beta_review` and default to `true` 😊 Let's go that approach!

Reply:
@joshdholtz I updated the PR and my manual tests succeed, but [one test](https://app.circleci.com/pipelines/github/fastlane/fastlane/4122/workflows/29cfe7af-0dd9-4f3e-a00a-1fdd252b08b2/jobs/79211) is not passing, and I cannot figure out why. Could you help me on that?

Reply:
To fix this test you could add `submit_beta_review: true,` in `distribute_options_non_localized` in `pilot/spec/build_manager_spec.rb`, but I suppose that it would be also good to add some tests for another set of options with `submit_beta_review: false`. What do you think @joshdholtz ?

Reply:
@joshdholtz can we merge this? Or should I add something else?